code/modules/tgs/v3210/commands.dm

```diff
@@ -42,5 +42,11 @@
 	var/datum/tgs_chat_command/stc = new command_type
 	var/datum/tgs_chat_user/user = new
 	user.friendly_name = sender
+
+	// Discord hack, fix the mention if it's only numbers (fuck you IRC trolls)
+	var/regex/discord_id_regex = regex(@"^[0-9]+$")
+	if(discord_id_regex.Find(sender))
```

Comment:
```suggestion
	if(findtext(sender, discord_id_regex))
```
Micro-optimization, this takes 2/3rds the time than the alternative:
<https://discord.com/channels/484170914754330625/487268744419344384/763179339512021002>

Reply:
bleeeeh why does no one ever review it when i release it

Reply:
Feel free to poke me if you want reviews.
How do we handle this, though?

Reply:
Already released https://github.com/tgstation/tgstation-server/pull/1137. I'll re-run the update action